integration/utils.go

```diff
@@ -68,3 +75,16 @@ func removeDockerImage(imageName string) error {
 	}
 	return nil
 }
+
+func cleanupRemoteHost(hostname string) error {
+	out, err := exec.Command("gcutil", "ssh", hostname, "sudo", "docker", "rm", "`docker ps -a -q`").CombinedOutput()
```

Comment:
this will try to remove the existing kubernetes containers and fail (which is fine). Maybe just ignore the error?